src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/ExternallyInteractable.java

```diff
@@ -0,0 +1,89 @@
+package io.github.thebusybiscuit.slimefun4.core.attributes;
+
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
+import org.bukkit.Location;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Implement this interface for any {@link SlimefunItem} to provide methods for
+ * external code to 'interact' with the item when placed as a block in the world.
+ *
+ * @author TheBusyBiscuit
```

Comment:
I do not remember writing this code...
#identitytheft

Reply:
Dr. CopyPasta strikes again xD